lib/gems/pending/util/miq-process.rb

```diff
@@ -237,31 +232,27 @@ def self.str_time_to_sec(time_str)
     (t[0].to_i * 3600) + (t[1].to_i * 60) + t[2].to_i
   end
 
+  # Suspend the process +pid+ using a SIGSTOP. If the process isn't running
+  # then this is no-op.
+  #
   def self.suspend_process(pid)
```

Comment:
Wasn't this something you were going to handle with `more-core-extensions`, or are you waiting to do that later?

Reply:
@NickLaMuro Updated, removed.